applications/teletype/src/teletype_util.erl

```diff
@@ -903,15 +903,15 @@ should_handle_notification_for_account(AccountId, ResellerId) ->
 
 -define(MOD_CONFIG_CAT(Key), <<(?NOTIFY_CONFIG_CAT)/binary, ".", Key/binary>>).
 
--spec is_notice_enabled(wh_json:object(), wh_json:object(), ne_binary()) -> boolean().
-is_notice_enabled(AccountJObj, ApiJObj, NoticeKey) ->
-    case {wh_json:get_value([<<"notifications">>
-                             ,NoticeKey
-                             ,<<"enabled">>
-                            ], AccountJObj)
-          ,wh_json:is_true(<<"Preview">>, ApiJObj, 'false')
-         }
-    of
+-spec is_notice_enabled(ne_binary(), wh_json:object(), ne_binary()) -> boolean().
+is_notice_enabled(AccountId, ApiJObj, NoticeKey) ->
+    AccountDb = wh_util:format_account_id(AccountId, 'encoded'),
```

Comment:
hon_rater uses wh_notify:system_alert that doesn't send account_id in the notify payload.
ne_binary() should be api_binary() and AccountId = 'undefined' can handled instead of waiting for couch error